doc/whats_new/v1.0.rst

```diff
@@ -35,13 +42,66 @@ Changelog
   and :class:`decomposition.MiniBatchSparsePCA` to be convex and match the referenced
   article. :pr:`19210` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+- |Fix| :class:`decomposition.FastICA` now validates input parameters in `fit`
+  instead of `__init__`.
+  :pr:`21432` by :user:`Hannah Bohle <hhnnhh>` and
+  :user:`Maren Westermann <marenwestermann>`.
```

Comment:
I would save the validate parameters fixes for 1.1, because they seem like big enough behavior changes.

Reply:
fair enough (I did not rebase yet :))